CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Change Log for FreeRTOS
 
+## New Features
+
+### Over the Air Update V1.2.0
+
+- Added feature to suspend and resume OTA Agent using new APIs provided. This allows OTA Agent to resume download after suspending it for any tasks like re-establishing MQTT connection. 
+- OTA Demo is updated to check for MQTT disconnect, suspend OTA operations and try reconnecting with exponential delay and jitter. After successful connection the demo resumes OTA Agent operations.
+- Added new OTA config for allowing downgrade and version update. This is provided for testing purpose and configuration is disabled by default.
+
+### Espressif
+
+- New Board: <b>ESP32-WROOM_32-SE</b> is now qualified by FreeRTOS. It was earlier supported in Preview mode before.
+- Support OTA data over HTTP together with BLE on ESP32 when SPIRAM is enabled.
+- Added ESP-IDF component for WiFi provisioning in SoftAP mode. This allows WiFi provisioning of devices through a web-server running on the device and a provisioning mobile application to enter WiFi credentials onto the device for provisioning. This requires use of lwIP as the networking stack.
+
+## Updates
+
+### FreeRTOS+POSIX Utils V1.2.0
+
+- Update UTILS_TimespecAdd POSIX utility function to support both signed and unsigned definitions of struct timespec.tv_sec. (Some implementations use unsigned definition of time_t to address the *2038* problem on 32-bit systems. ) This change is backwards compatible.
```

Comment:
Some implementations use unsigned definition of time_t --> Some implementations use unsigned definition of struct timespec.tv_sec

Reply:
Done